README.md

```diff
@@ -1 +1,36 @@
 # hw01
+
+My name is Leyi Kang. I am a master student studying psychology in the Social Sciences Division. I just graduated from the University of Rochester this May, and I look forward to delving deeper into psychology research on child and adolescent psychopathology at UChicago. My long-term goal is to apply for either clinical or developmental psychology PhD after I graduate from here. 
+
+
+# Header 1
+
+## Header 2
+
+### Header 3
+
+#### Header 4
+
+*This sentence will be italic.* or _This will also be italic._
+
+**This sentence will be bold.** or __This will also be bold.__
+
+__This will be *both*.__
+
+* Unordered List
+* Colors
+  + blue
+  + red
```

Comment:
I think I recognize this bit from one of the reference articles? I think we were supposed to use the extra text options (headers, italics, etc.) to continue with our introduction.

Reply:
That's correct! There seemed to be some confusion here, but that's alright for now.